doc/development.rst

```diff
@@ -326,6 +328,10 @@ A root state is a state that has no dependencies.
         graph_strategy.transition('B')  #  returns: []
         graph_strategy.transition('B', via=['A2'])  #  returns: ['Root', 'A2', 'B']
 
+In case shortcuts can be followed after a specific state was reached
+introduce an instance variable and decide on that which path to follow. This
+should be done in a `pytest fixture <https://docs.pytest.org/en/latest/fixture.html>`_.
+
```

Comment:
Why do you recommend to move some system state and logic from the Strategy to a fixture? Or am I misunderstanding this?

Reply:
Sorry, that was phrased improperly. The idea is to introduce system state to the Graph Strategy and use that in the fixture to cut corners with `transition('my_tate', via=..)`. I will rephrase this.